package.json

```diff
@@ -22,11 +22,11 @@
     "build": "rm -rf lib/** && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json && npm run format-check && npm run api-extractor && npm run generate-docs && npm run generate-notices && vite build",
     "prepare": "husky install",
     "format-check": "prettier --check . && eslint .",
-    "format-fix": "prettier --write . && eslint . --fix",
+    "format-fix": "prettier --write .",
     "api-extractor": "api-extractor run --local --verbose",
     "generate-docs": "api-documenter markdown --input-folder temp --output-folder docs && rm -rf temp",
     "prepublishOnly": "npm run build",
-    "test": "jest && playwright test",
+    "test": "jest && playwright install --with-deps && playwright test",
```

Comment:
Why are we doing this? You shouldnt need install every single time the test is called. playwright test will ask you to install the package if you're missing anything.

Reply:
<img width="1781" alt="Screenshot 2023-09-21 at 3 18 31 PM" src="https://github.com/yext/analytics/assets/39343323/62c0ed94-8f80-4155-8639-e1594e076385">

Reply:
Added this to npm run build then.